src/main/java/io/gatling/jenkins/GatlingBuildAction.java

```diff
@@ -41,7 +41,7 @@
  */
 public class GatlingBuildAction implements Action, SimpleBuildStep.LastBuildAction {
 
-  private final Run<?, ?> run;
+  private final Run<?, ?> run; // TODO make transient, implement RunAction2
```

Comment:
This field could come back to bite you under certain memory pressure conditions.